scalableminds/webknossos
# Refactor Backend: Merge TimeSpanSQL and TimeSpan

### URL of deployed dev instance (used for testing):
- https://refactorbackendtimespans.webknossos.xyz

### Steps to test:
- create task, trace some, see that tracing time shows correctly in statistics→time tracking

### Issues:
- contributes to #2521

------
